shared/naturalcrit/codeEditor/codeEditor.jsx

```diff
@@ -16,8 +16,24 @@ if(typeof navigator !== 'undefined'){
 	require('codemirror/mode/javascript/javascript.js');
 
 	//Addons
+	//Code folding
 	require('codemirror/addon/fold/foldcode.js');
 	require('codemirror/addon/fold/foldgutter.js');
+	//Search and replace
+	require('codemirror/addon/search/search.js');
+	require('codemirror/addon/search/searchcursor.js');
+	require('codemirror/addon/search/jump-to-line.js');
+	require('codemirror/addon/search/match-highlighter.js');
+	require('codemirror/addon/search/matchesonscrollbar.js');
+	require('codemirror/addon/dialog/dialog.js');
+	//Trailing space highlighting
+	require('codemirror/addon/edit/trailingspace.js');
+	//Active line highlighting
+	require('codemirror/addon/selection/active-line.js');
```

Comment:
active line is getting cleared as soon as you type. Probably related to our `highlightcustommarkdown()` function clearing out line styles? Unless that's the intended behavior.

Reply:
Yes I discovered that last night and hadn't committed the fix. I'm not sure what happened but I think a merge from master undid the change I made to fix that, because the change was no longer in my codebase.

Reply:
Fixed!